plugins/serverless/src/tasks/run.ts

```diff
@@ -9,7 +9,7 @@ import waitPort from 'wait-port'
 export default class ServerlessRun extends Task<typeof ServerlessSchema> {
   static description = 'Run serverless functions locally'
 
-  static defaultOptions: ServerlessOptions = {
```

Comment:
**question:** Why are you removing the `ServerlessOptions` type here? Is it in preparation for the Zod PR where this will no longer be needed?

Reply:
No, it's because it throws a type error if we keep it as we don't specify default options for [`awsAccountId`, `systemCode`, and `region`](https://github.com/Financial-Times/dotcom-tool-kit/blob/9f3c4be57255fd39b87aed801218cc00c7769c65/lib/types/src/schema/serverless.ts#L4-L6), which are required options but must be specified by the user. I was allowing it to fall back to the default type for `defaultOptions`, [`Record<string, unknown>`](https://github.com/Financial-Times/dotcom-tool-kit/blob/9f3c4be57255fd39b87aed801218cc00c7769c65/lib/types/src/index.ts#L151), but I think it would be better to use `Partial<ServerlessOptions>` actually so that we can type check the fields we _are_ setting!

Reply:
Nice!